src/main.rs

```diff
@@ -0,0 +1,32 @@
+fn main() {
+    println!("Bem vindo ao TODO List");
+    loop{
+        if !create_todo(){
+            break
+        }   
+    }
```

Comment:
Uma forma mais simples de escrever esse loop seria usando um `while`:
```suggestion
    while create_todo() {}
```